vllm/engine/async_llm_engine.py

```diff
@@ -1189,7 +1189,7 @@ async def get_parallel_config(self) -> ParallelConfig:
         """Get the parallel configuration of the vLLM engine."""
         return self.engine.get_parallel_config()
 
-    async def get_decoding_config(self) -> DecodingConfig:
+    async def get_decoding_config(self) -> StructuredOutputConfig:
```

Comment:
Maybe we should rename this method?

Reply:
This is in V0 code, so I'm less prone to edit more than need be, given that we are removing this soon anw

Reply:
I'm happy leaving V0 code alone, is this method a protocol that must be shared between V0 and V1 though?

Reply:
Apparently this does exists in the v1 protocol, but yet to be implemented 🤷